### DISCUSSION TITLE
useEventTarget,在拼音输入的过程中，是否可以不变更value的值，直到拼音输入结束再发生变更

### DISCUSSION THREAD

**Discussion by candy4290** [alibaba/hooks#2208] [Ideas]
我这里的场景，很多时候不希望在拼音输入的过程中就触发value的变更。
但是我不确定这种场景是否是一个常见的场景。
如果是的话，我有个想法。提供一个option
参数-说明-类型-默认值
ignoredInComosition-在composition过程中不触发value变更-默认值false。

实现思路：在ignoredInComosition为true时。通过判断compostion所处的阶段来决定是否触发value的更新。
如果可以这样做的话，我将尝试提交一个PR，期待你们的反馈。

**Comment by liuyib** [comment]
确实有这个使用场景，一个自动搜索的输入框，想要输入完汉字后再触发 `onChange` 搜索。

来吧，PR

**Reply by candy4290** [reply]
好的~

**Comment by candy4290** [comment]
我的方法需要，改动相对多点。不是个好办法，我先关闭了这个讨论吧~~~
